### ISSUE TITLE
Fedora 34 copr support

### ISSUE THREAD

**Issue by makarov-roman** [daniruiz/flat-remix#169]
Hello!
Can you please add a build for f34 in copr https://copr.fedorainfracloud.org/coprs/daniruiz/flat-remix/ ?

**Comment by daniruiz** [comment]
Done ;)

**Comment by daniruiz** [comment]
wait, it's failing :(
I'll check it again, thanks

**Comment by rivenirvana** [comment]
Any updates on this? An issue for the other packages in the COPR repo, as well.

**Comment by daniruiz** [comment]
Unfortunatelly no :(
I don't understand why the packages are not building for Fedora 34 but they do for Fedora 33. I didn't have enough time to fix it anyway

You can use the package provided by the official fedora repos (flat-remix-icon-theme). I'll probably take some ideas from that one to make mine work :D
![image](https://user-images.githubusercontent.com/12565871/121866125-efb04c00-ccfe-11eb-9feb-8ae004de7ece.png)

**Comment by rivenirvana** [comment]
I understand. Although, whenever you're able to, could you rebuild the packages again so that I and others interested can check the build logs in COPR and see what's wrong? The logs for failed chroots seem to have been deleted now, returning 404s.

**Comment by rivenirvana** [comment]
I checked the latest build's logs and found out the problem. Seems that `cmake` is not preinstalled with the chroot indicated by the error `/var/tmp/rpm-tmp.lQuHW2: line 37: /usr/bin/make: No such file or directory`. The spec only needs to specify that `cmake` is a build requirement to fix it. I used the same spec with `BuildRequires:  cmake` in a test copr and the build succeeds for Fedora 34.

Build link: https://copr.fedorainfracloud.org/coprs/rivenirvana/test/build/2285446/
